aea/cli/registry/fetch.py

```diff
@@ -44,6 +44,7 @@ def fetch_agent(ctx: Context, public_id: PublicId) -> None:
     target_folder = os.path.join(ctx.cwd, name)
     os.makedirs(target_folder, exist_ok=True)
 
+    click.echo('Fetching dependencies...')
```

Comment:
I notice, that in some places we use click built in print functionality and in other places the logger.info. Can we unify this as part of this PR and use `click.echo` for all `logger.info`? @marcofavorito thoughts? I think `click.echo` might be preferable as it means the `info` bit is missing from the output.

Reply:
I would prefer `click.echo` for messages that should reach user. Logging probably sould be used for dev related output.

Reply:
Ok, so please apply `click.echo` throughout where `logger.info` is used atm. `logger.debug` of course remains.